ai/research/hillerfit-benchmark-leaderboard/benchmark-scoring-walkthrough.html

```
@@ -0,0 +1,339 @@
+<h2 id="big-picture">1. The big picture</h2>
+<p>Benchmark scoring is <strong>two parallel systems</strong> that meet on the leaderboard:</p>
+<ul>
+  <li><strong>Ranking</strong> — the <em>online</em> algorithm (place-based, golf-style: 1st&nbsp;=&nbsp;1&nbsp;pt, lowest total wins) decides who is above whom.</li>
+  <li><strong>Tier context</strong> — the threshold tables you edit on the tiers page turn every raw result into a tier (1…maxTier, or 0.5), tiers aggregate into category scores and an overall <code>x/100</code> with a rating band. This is <em>additive display data</em>, not the ranking — unless the competition still uses the legacy <code>absolute_tier</code> algorithm.</li>
+</ul>
+
+<div class="card">
+<pre class="mermaid">
+flowchart LR
+    subgraph SETUP["Organizer setup"]
+        A["tiers.tsx<br/>threshold editor"] --> B["saveBenchmarkScoringTiersFn"]
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)